### DISCUSSION TITLE
More Compact Layout

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#1009] [Ideas]
<h1>Request</h1>

Please provide a more compact layout option, or provide more customization options to create a more compact layout. This would allow the user to see more posts on the screen at once.

<h1>Examples</h1>

<h2>Infinity</h2>

This is the most compact that I could get it without losing flair, post info, or awards. Choosing Compact Layout and making title font smaller.

<img src="https://i.imgur.com/KqGgPMR.jpeg" width="50%" height="50%">

<img src="https://i.imgur.com/KFOiLJu.jpeg" width="50%" height="50%">

<b><font size="larger">Hiding Toolbar by Default</font></b>

This makes you lose the ability to see the number of votes and comments, but it is more compact.

<img src="https://i.imgur.com/yNVSI09.jpeg" width="50%" height="50%">

<img src="https://i.imgur.com/ak1BEzC.jpeg" width="50%" height="50%">

<h2>Slide</h2>

Heavily customized. Slide provides a lot of options for customization.

<img src="https://i.imgur.com/emXkm9s.jpeg" width="50%" height="50%">

<h2>RedReader</h2>

Removed "Inline image previews" in settings, the rest is default.

<img src="https://i.imgur.com/eVgZznP.jpeg" width="50%" height="50%">

<img src="https://i.imgur.com/KU6swM6.jpeg" width="50%" height="50%">

<h1>Suggestion</h1>

When hiding toolbar by default, it should move the number of votes and comments somewhere else, or have it somewhere else by default.

Maybe adding 1 more level of smaller font size option would help as well.

For **Compact Layout 2**, maybe the Post info should be restructured. The way that the username, date/time, post type, post flair, and award are portrayed takes up quite a bit of space.

**Comment by Wladefant** [comment]
Thank you for your suggestion!

**Comment by Wladefant** [comment]
Similar to #51

**Comment by Pentaphon** [comment]
I also think we need a more compact layout than the current compact layout. 

We need a layout that is half the size of the current compact layout.
